### PR TITLE
Added Booting state

### PR DESCRIPTION
**Pull request by dylandesrosier**
Additional check on boot to avoid double subscription to EthLogs

### CODE REVIEW

File: watchdog.go [hunk 1]
```diff
@@ -70,6 +72,7 @@ func NewWatchdog(
 	w := &Watchdog{
 		PathToChildExecutable: pathToChildExecutable,
 		Args:                     cpOfArgs,
+		Starting:                  make(chan bool, 1),
```
Comment (ann-kilzer) [inline watchdog.go]: nit: run gofmt here. 

What editor do you use? 
[VS code has a go plugin.](https://robots.thoughtbot.com/writing-go-in-vim)
[Using go with emacs](http://dominik.honnef.co/posts/2013/03/writing_go_in_emacs/)
[Using go with vim](https://robots.thoughtbot.com/writing-go-in-vim)

**Comment by ann-kilzer** [comment]
looking good, just be sure to always run go fmt :)

(Maybe I can figure out a way to automagically make github do this. Until then, I linked to some plugins for popular code editors)

File: watchdog.go (hunk 1 above)
Comment (dylandesrosier) [reply watchdog.go]: Atom, I downloaded one and turned on auto format but didn't turn on the plugin.. Fixed

**Comment by ann-kilzer** [comment]
LGTM⭐

**Pull request merged**